introduction.md

```diff
@@ -1 +1,4 @@
-On my way to becoming a GitHub Master. I love Ruby and JavaScript.
+On my way to becoming a GitHub Master. I love Ruby and JavaScript and Git.
+list:
+* git;
+* js;
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?